nc/source/engine/Engine.cpp

```diff
@@ -255,6 +255,7 @@ namespace nc::core
             // No UI for us until we tackle integrating IMGUI with Vulkan
         #else
             m_window.BindGraphicsOnResizeCallback(std::bind(graphics::Graphics::OnResize, &m_graphics, _1, _2, _3, _4, _5));
+            m_window.BindGraphicsSetClearColorCallback(std::bind(graphics::Graphics::SetClearColor, &m_graphics, _1));
```

Comment:
do we need this much ado around the clear color? Any reason it can't be set as a const once in some file and forgotten about?